common/changes/office-ui-fabric-react/edwl-2018-09-fixTypedefTsLint_2018-09-21-16-26.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Remove typedef tslint override",
+      "type": "minor"
```

Comment:
Should this be `none` or `minor`?

Reply:
I'd definitely recommend minor at the least.

Reply:
I'd keep at minor. There could be unintended breaks here, better be safe.